common/gratia/common/condor_ce.py

```diff
@@ -362,9 +367,14 @@ def queryJob(jobid):
     certinfo = {}
     if 'x509UserProxyVOName' in info:
         certinfo["VO"] = info['x509UserProxyVOName']
+    elif 'AuthTokenIssuer' in info:
+        certinfo["VO"] = info['AuthTokenIssuer']
     if 'x509userproxysubject' in info:
         certinfo['DN'] = info['x509userproxysubject']
+    elif 'AuthTokenSubject' in info:
+        certinfo['DN'] = info['AuthTokenSubject']
     if 'x509UserProxyFirstFQAN' in info:
         certinfo['FQAN'] = info['x509UserProxyFirstFQAN']
+    # intentionally omitting AuthTokenIssuer equivalent for FQAN; see PR #96
```

Comment:
```suggestion
    # FQAN is used as a backup for DN/VO in the certinfo to populate
    # VOName/ReportableVOName in the usage record, respectively.
    # Since DN/VO are always populated in the AuthToken case, we are
    # intentionally omitting AuthTokenIssuer equivalent for FQAN.
    # See https://github.com/opensciencegrid/gratia-probe/pull/96#discussion_r651148685
```

Let's describe the why in addition to the what. Also a direct link to the discussion/comment about dropping this is preferable, especially since there's been a few other discussions in this PR.